src/corona_hakab_model/default_configuration/medical_state_machine.json

```diff
@@ -0,0 +1,179 @@
+{
+  "medical_states": {
+    "Susceptible": {
+      "susceptible": true,
+      "contagiousness": 0,
+      "test_willingness": 0.01,
+      "detectable": false,
+      "terminal": true
+    },
+    "Latent": {
+      "susceptible": false,
+      "contagiousness": 0,
+      "test_willingness": 0.01,
+      "detectable": false,
+      "stochastic": true
+    },
+    "Silent": {
+      "susceptible": false,
+      "contagiousness": 0.3,
+      "test_willingness": 0.01,
+      "detectable": true,
+      "stochastic": true
+    },
+    "Symptomatic": {
+      "susceptible": false,
+      "contagiousness": 0.75,
+      "test_willingness": 0.6,
+      "detectable": true,
+      "stochastic": true
+    },
+    "Asymptomatic": {
+      "susceptible": false,
+      "contagiousness": 0.25,
+      "test_willingness": 0.01,
+      "detectable": true,
+      "stochastic": true
+    },
+    "Hospitalized": {
+      "susceptible": false,
+      "contagiousness": 0.0,
+      "test_willingness": 0.9,
+      "detectable": true,
+      "stochastic": true
+    },
+    "ICU": {
+      "susceptible": false,
+      "contagiousness": 0.0,
+      "test_willingness": 1.0,
+      "detectable": true,
+      "stochastic": true
```

Comment:
SInce a stateis either Terminal, Stochastic or in the future AgentAware, why not write "medical_state_type" and have it be filled with "Stochastir", "Terminal" or "AgentAware"?